From 9ad4827f8d24dee9b46f596699e819b14944333e Mon Sep 17 00:00:00 2001
From: Matt Porter <mporter@linaro.org>
Date: Wed, 5 Feb 2014 14:01:38 -0500
Subject: [PATCH 3/5] iio: enable selection and build of pulse drivers

Add the pulse driver subdirectory when configuring and building
IIO.

Signed-off-by: Matt Porter <mporter@linaro.org>
Signed-off-by: Stephen Arnold <sarnold@vctlabs.com>
---
 drivers/iio/Kconfig  | 1 +
 drivers/iio/Makefile | 1 +
 2 files changed, 2 insertions(+)

diff --git a/drivers/iio/Kconfig b/drivers/iio/Kconfig
index 6743b18194fb..bb652ed3af56 100644
--- a/drivers/iio/Kconfig
+++ b/drivers/iio/Kconfig
@@ -89,6 +89,7 @@ endif #IIO_TRIGGER
 source "drivers/iio/potentiometer/Kconfig"
 source "drivers/iio/pressure/Kconfig"
 source "drivers/iio/proximity/Kconfig"
+source "drivers/iio/pulse/Kconfig"
 source "drivers/iio/temperature/Kconfig"

 endif # IIO
diff --git a/drivers/iio/Makefile b/drivers/iio/Makefile
index 87e4c4369e2f..9564e7338b09 100644
--- a/drivers/iio/Makefile
+++ b/drivers/iio/Makefile
@@ -31,5 +31,6 @@ obj-y += orientation/
 obj-y += potentiometer/
 obj-y += pressure/
 obj-y += proximity/
+obj-y += pulse/
 obj-y += temperature/
 obj-y += trigger/
--
2.12.2
